app/views/users/_login_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<center>
+  <div class="card" style="width: 20rem;">
+    <h2 class="card-header">Log in</h2>
+    <% controller.class == Admin::SessionsController ? url = admin_sign_in_path : url = user_session_path %>
+    <%= form_for(resource, as: resource_name, url: url) do |f| %>
```

Comment:
Lets move to form_with

Reply:
https://trello.com/c/QUYxx5il/13-formwith-instead-of-formfor-in-devise-views